jenkins/ci.suse.de/ardana-job.yaml

```diff
@@ -177,14 +178,22 @@
 
           cat ardana_net_vars.yml
 
-          ansible-playbook -vvv -i hosts ssh-keys.yml
-          ansible-playbook -vvv -i hosts -e "build_url=$BUILD_URL" \
-                                         -e "cloudsource=${cloudsource}" repositories.yml
-          ansible-playbook -vvv -i hosts -e "deployer_floating_ip=$DEPLOYER_IP" \
-                                         -e "deployer_model=${model}" \
-                                         -e "tempest_run_filter=${tempest_run_filter}" \
-                                         init.yml
-
+          ansible-playbook -v -i hosts ssh-keys.yml
+          ansible-playbook -v -i hosts -e "build_url=$BUILD_URL" \
+                                       -e "cloudsource=${cloudsource}" repositories.yml
+          ssh="ssh $sshargs ardana@$DEPLOYER_IP"
+          verification_mktemp_dir=$($ssh "mktemp -d /tmp/ardana-job-rpm-verification.XXXXXXXX")
+          ansible-playbook -v -i hosts -e "deployer_floating_ip=$DEPLOYER_IP" \
+                                       -e "deployer_model=${model}" \
+                                       -e "verification_mktemp_dir=$verification_mktemp_dir" \
+                                       init.yml
+          # Run site.yml outside ansible for output streaming
+          $ssh "cd ~/scratch/ansible/next/ardana/ansible ; \
+               ansible-playbook -vvv -i hosts/verb_hosts site.yml"
```

Comment:
[Meta comment: I can't reply to [the previous discussion on this hunk](https://github.com/SUSE-Cloud/automation/pull/2469#discussion_r179911952) because GitHub has hidden it forever.  It's not even showing collapsed outdated diffs which can be reexpanded.  So I just have to rely on my memory of what I wrote and where I wrote it.  This is pretty much the worst aspect of GitHub code review, and a total show-stopper IMHO.  See https://github.com/isaacs/github/issues/997, https://github.com/isaacs/github/issues/998, and https://github.com/isaacs/github/issues/999.]

OK, so you are relying on `set -e` here.  I'll avoid a general rant about the many problems of `set -e` but the main problem in this case is the obfuscation - it's not clear at a glance that this will interrupt execution, except for people who at some point spotted the `set -ex` at the top *and* know what that does *and* remembered that it's there.  So I'd instead recommend wrapping the ssh in an `if`, or checking `$?` after.  That also gives the opportunity to output a `site.yml failed; aborting` summary message to make it really clear what happened to the execution flow.

One downside of bailing out immediately here is that it skips the rpm checks which can still be useful even when `site.yml` fails.  OTOH a slight upside is that it makes the `site.yml` failure a bit easier to locate since it's always at the end of the console log.  I think technically the previous approach was more correct but I'm OK with sacrificing that for convenience.